### PR TITLE
chore(circleci): Bump timeout from 2m to 4m in indexer lint

### PR DESCRIPTION
**Pull request by roninjin10**
- remove timeout from lint
https://github.com/ethereum-optimism/optimism/pull/6808#discussion_r1295231513

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #6810** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6810" target="_blank">Graphite</a>  👈
    * **PR #6808** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6808" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/6810?utm_source=stack-comment).

**Review by tynes — APPROVED** [review]

**Review by trianglesphere — APPROVED** [review]
should probably merge directly into develop

File: .circleci/config.yml [hunk 1]
```diff
@@ -904,7 +904,7 @@ jobs:
           patterns: indexer
       - run:
           name: Lint
-          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 2m -e "errors.As" -e "errors.Is" ./...
+          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint -e "errors.As" -e "errors.Is" ./...
```
Comment (trianglesphere) [inline .circleci/config.yml]: sorry, I'm stupid, the golangci-lint default timeout is 1m (not 10m like golang test). We should probably bump it to like 4m

Comment (roninjin10) [reply .circleci/config.yml]:
```suggestion
          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint - --timeout 4m -e "errors.As" -e "errors.Is" ./...
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -904,7 +904,7 @@ jobs:
           patterns: indexer
       - run:
           name: Lint
-          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 2m -e "errors.As" -e "errors.Is" ./...
+          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint - --timeout 4m -e "errors.As" -e "errors.Is" ./...
```
Comment (roninjin10) [inline .circleci/config.yml]:
```suggestion
          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 4m -e "errors.As" -e "errors.Is" ./...
```

**Review by felipe-op — APPROVED** [review]

**Comment by tynes** [comment]
This looks like it needs a patch still where the timeout flag is added back in

File: .circleci/config.yml (hunk 1 above)
Comment (roninjin10) [reply .circleci/config.yml]:
```suggestion
          command: golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint --timeout 2m -e "errors.As" -e "errors.Is" ./...
```
@tynes is correct I force pushed away this change

**Pull request merged**